spec/person_spec.rb

```diff
@@ -0,0 +1,36 @@
+require_relative '../person'
+
+describe Person do
```

Comment:
- [ Required ] Please consider adding a unit test for the `add_rental` method in the `Person` class, It would be better if the method was tested to make sure that it works as expected. As stated in the project requirements all the methods in a given class should be tested, please see the screenshot below for more details on this and rectify it.

## Project requirements
![](https://user-images.githubusercontent.com/16392046/225440383-4dcd874f-51d6-4f82-9560-214130e7f424.png)